users-service/features/support/env.ls

```diff
@@ -8,7 +8,7 @@ require! {
 db = null
 get-db = (done) ->
   return done db if db
-  MongoClient.connect "mongodb://localhost:27017/space-tweet-users-test", N (mongo-db) ->
+  MongoClient.connect "mongodb://localhost:27017/space-tweet-db-test", N (mongo-db) ->
```

Comment:
We're using the same database for the whole application, no? Unless we want to test in separate databases for tweets and users.